### PR TITLE
Docs: ImageMagick vulnerability CVE-2016–3714

### PR DESCRIPTION
**Pull request by jaredbeck**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -14,6 +14,9 @@ You can chat with us using Gitter:
 
 * [Bundler](http://gembundler.com)
 * [ImageMagick](http://www.imagemagick.org/script/install-source.php)
+  * :warning: Warning: ImageMagick currently has a serious security vulnerability, CVE-2016–3714. After installing, you must disable certain features in ImageMagick's policy configuration. Please see the following for details:
+    * https://imagetragick.com/
+    * https://github.com/refinery/refinerycms/issues/3171
```
Comment (simi) [inline readme.md]: I think you can remove this line. :scissors: That first link is enough IMHO.

**Comment by simi** [comment]
:+1: , but leaving open for @parndt review

File: readme.md (hunk 1 above)
Comment (parndt) [inline readme.md]: I completely agree with this review

**Comment by jaredbeck** [comment]
> I think you can remove this line.

Removed, thanks.

**Comment by bricesanchez** [comment]
@jaredbeck Could you squash you commits ?

**Comment by simi** [comment]
@bricesanchez actually squash maybe will not be needed. I think GitHub can squash on its own now.

**Comment by bricesanchez** [comment]
@simi it looks like to be an enterprise feature : https://enterprise.github.com/releases/2.6.0#squash-commits

**Comment by simi** [comment]
I think that's open for all now (https://github.com/blog/2141-squash-your-commits), but someone needs to enable it in settings.

Anyway I'm ok with this change now. I can squash and merge later today.

**Comment by bricesanchez** [comment]
@simi Nice!
@parndt Could you add "Allow squash merging" in Github settings : https://github.com/blog/2141-squash-your-commits#whats-changing?

**Comment by simi** [comment]
:heart:

**Comment by parndt** [comment]
It was already enabled

**Pull request merged**